kernel/virtio/boot.S

```diff
@@ -141,42 +140,58 @@ END(_start)
 
 .data
 .align 64
-.globl cpu_gdt64
-cpu_gdt64:
+gdt64:
 	.quad 0x0000000000000000
-	.quad 0x00af9b000000ffff	/* 64bit CS		*/
+	.quad GDT_DESC_CODE_VAL		/* 64bit CS		*/
```

Comment:
You are right, sorry for the confusion. The field changed is type which for a code segment means:
```
before: 0x00af9b000000ffff -> type = 0xb (Execute-Only, accessed)
now: 	0x00af99000000ffff -> type = 0x9 (Execute/Read, accessed)
```